--- nginx.conf.web.http.default_ORIG	2017-02-03 16:15:02.299468690 +0100
+++ nginx.conf.web.http.default	2017-02-03 16:56:32.295986131 +0100
@@ -110,6 +110,7 @@
         # Fudge inter-mailbox redirects (kludge)
         proxy_redirect http://$relhost/ http://$http_host/;
     }
+    location ^~ /.well-known { root /opt/zimbra/data/nginx/html; }
 
     #For long polling of Microsoft ActiveSync
     location ^~ /Microsoft-Server-ActiveSync
@@ -357,7 +358,7 @@
     #     proxy_redirect http://$relhost/ http://$http_host/;
     # }
     
-    location ~* /(service|principals|dav|\.well-known|home|octopus|shf|user|certauth|spnegoauth|(zimbra/home)|(zimbra/user))/
+    location ~* /(service|principals|dav|home|octopus|shf|user|certauth|spnegoauth|(zimbra/home)|(zimbra/user))/
     {
         # Begin stray redirect hack
         #
